src/app/organizations/collection/collection.component.html

```diff
@@ -101,18 +104,28 @@ <h3 *ngIf="Dockstore.FEATURES.enableNotebooks">Workflows, Notebooks, and Tools</
             <h3 *ngIf="!Dockstore.FEATURES.enableNotebooks">Workflows and Tools</h3>
 
             <div data-cy="collectionEntry" *ngFor="let entry of collection?.entries">
-              <mat-card class="pb-3">
+              <mat-card
+                *ngIf="
+                  (entry.entryType !== entryType.Notebook && !Dockstore.FEATURES.enableNotebooks) || Dockstore.FEATURES.enableNotebooks
+                "
+                class="pb-3"
+              >
                 <a
                   [routerLink]="[
-                    '/' + (entry.entryType === 'tool' ? 'container' : 'workflow') + 's/',
+                    '/' +
+                      (entry.entryType === entryType.Tool
+                        ? 'container'
+                        : entry.entryType === entryType.Workflow
```

Comment:
IDE error on `entryType.Workflow` -- it doesn't exist. Actually, we have a pipe for this src/app/entry/router-link.pipe.ts

Reply:
> Actually, we have a pipe for this src/app/entry/router-link.pipe.ts

Mm previously considered that, unfortunately it takes in a type of `Workflow` to compute the url but here we're dealing with type `CollectionEntry`. But maybe its a good idea to add support for both, I'll update the pipe to take in `entry: Workflow | CollectionEntry` and handle both cases